coordinator/src/node/expired_positions.rs

```diff
@@ -0,0 +1,104 @@
+use crate::db;
+use crate::node::Node;
+use crate::node::TradeAction;
+use crate::position::models::Position;
+use crate::position::models::PositionState;
+use hex::FromHex;
+use lightning::ln::PaymentHash;
+use time::OffsetDateTime;
+use trade::bitmex_client::BitmexClient;
+
+pub async fn close(node: Node) {
+    let mut conn = match node.pool.get() {
+        Ok(conn) => conn,
+        Err(e) => {
+            tracing::error!("Failed to get pool connection. Error: {e:?}");
+            return;
+        }
+    };
+
+    let positions = match db::positions::Position::get_all_open_positions(&mut conn) {
+        Ok(positions) => positions,
+        Err(e) => {
+            tracing::error!("Failed to get positions. Error: {e:?}");
+            return;
+        }
+    };
+
+    let positions = positions
+        .into_iter()
+        .filter(|p| {
+            p.position_state == PositionState::Open
+                && OffsetDateTime::now_utc().ge(&p.expiry_timestamp)
+        })
+        .collect::<Vec<Position>>();
```

Comment:
Nit: 
- `positions` should already only include `open` positions.
- it might be more efficient to include the `expiry` check in the sql query